build/ios/cross

```diff
@@ -43,8 +43,8 @@ export LDFLAGS="-arch ${ARCH} $MINVERSION -isysroot $(xcrun -sdk ${PLATFORM} --s
 
 export pkg_configure_flags="$EXTRA_CONFIG"
 export pkg_prefix="$DESTDIR"
-export pkg_cross="ios"
-export pkg_cross_arch="${PLATFORM}/${ARCH}"
+export pkg_cross="${PLATFORM}"
+export pkg_cross_arch="${ARCH}"
```

Comment:
Adjust platform and arch so that we end up installing libs where we would like them to be. See above comment regarding the fact that the semantic of `pkg_cross` is now changed.